screens/ImagesScreen/RenderImagePairs.jsx

```diff
@@ -17,17 +17,29 @@ const RenderImagePairs = changeProfileImage => {
   const {arrayImages} = useContext(GlobalDataContext);
   const maximum = 4;
   const minimum = 1;
+  const [length1, setLength1] = useState(1);
+  const [length2, setLength2] = useState(4);
```

Comment:
length of what? what maximum and minimum? try to choose better names

Reply:
I will fix it.